spring-cloud-gcp-storage/src/test/java/org/springframework/cloud/gcp/storage/GoogleStorageTests.java

```diff
@@ -260,6 +261,30 @@ public void testCreateRelativeSubdirs() throws IOException {
 		Assert.assertEquals("gs://test-spring/t1/r1/relative.png", relative.getURI().toString());
 	}
 
+	@Test
+	public void nullSignedUrlForNullBlob() throws IOException {
+		String location = "gs://test-spring/t1/test.png";
+		Storage storage = Mockito.mock(Storage.class);
+		GoogleStorageResourceObject resource = new GoogleStorageResourceObject(storage,
+				location);
+		Mockito.when(storage.get(Mockito.any(BlobId.class))).thenReturn(null);
+		Assert.assertNull(resource.createSignedUrl(TimeUnit.DAYS, 1));
+	}
+
+	@Test
+	public void signedUrlFunctionCalled() throws IOException {
+		String location = "gs://test-spring/t1/test.png";
+		Storage storage = Mockito.mock(Storage.class);
+		Blob blob = Mockito.mock(Blob.class);
+		Mockito.when(blob.getBucket()).thenReturn("fakeBucket");
+		Mockito.when(blob.getName()).thenReturn("fakeObject");
+		GoogleStorageResourceObject resource = new GoogleStorageResourceObject(storage, location);
+		Mockito.when(storage.get(Mockito.any(BlobId.class))).thenReturn(blob);
+		resource.createSignedUrl(TimeUnit.DAYS, 1L);
+		Mockito.verify(storage, Mockito.times(1))
```

Comment:
nit: we've been using static imports in tests, you might want to do the same here for consistency.